src/assets/data/edition/series/2/section/2a/m267/source-description.json

```diff
@@ -0,0 +1,279 @@
+                ],
+                "writingInstruments": {
+                    "main": "schwarze Tinte",
+                    "secondary": ["Bleistift", "grüner Buntstift"]
+                },
+                "titles": [],
+                "dates": [
+                    "<em>Kinderstücke | (Herbst 1924) </em>mit grünem Buntstift auf Bl. 1<sup>r</sup> oben rechts"
+                ],
+                "paginations": [],
+                "measureNumbers": [],
+                "instrumentations": [],
+                "annotations": [],
+                "contents": [
+                    {
+                        "item": "M 267",
+                        "itemLinkTo": {
+                            "complexId": "m267",
+                            "sheetId": "M_267"
+                        },
+                        "itemDescription": "(Kinderstück für Klavier M 267)",
+                        "folios": [
+                            {
+                                "folio": "1r",
+                                "folioLinkTo": "",
+                                "folioDescription": "",
+                                "systemGroups": [
+                                    [
+                                        {
+                                            "system": "1–2",
+                                            "measure": "1–4",
+                                            "linkTo": ""
+                                        }
+                                    ],
+                                    [
+                                        {
+                                            "system": "4–5",
+                                            "measure": "5–7",
+                                            "linkTo": ""
+                                        }
+                                    ],
+                                    [
+                                        {
+                                            "system": "7–8",
+                                            "measure": "8–10",
+                                            "linkTo": ""
+                                        }
+                                    ],
+                                    [
+                                        {
+                                            "system": "10–11",
+                                            "measure": "11–13",
+                                            "linkTo": ""
+                                        }
+                                    ],
+                                    [
+                                        {
+                                            "system": "13–14",
+                                            "measure": "14–15",
+                                            "linkTo": ""
+                                        }
+                                    ]
```

Comment:
Fehlen hier noch T. 16–17? Diese sind in SVG von TF1, aber nicht hier in Inhaltsübersicht. Vermutlich auf Bl. 1v? Ansonsten müsste in Zustandsbeschreibung auch noch Bl. 1v als unbeschrieben erwähnt werden?